tests/test_llms.py

```diff
@@ -159,6 +166,199 @@ def test_pickling(self, tmp_path: pathlib.Path) -> None:
         assert llm.router.deployment_names == rehydrated_llm.router.deployment_names
 
 
+class TestOpenAIBatchLLMModel:
+    @pytest.fixture(scope="class")
+    def config(self, request) -> dict[str, Any]:
+        model_name = request.param
+        return {
+            "model": model_name,
+            "temperature": 0.0,
+            "max_tokens": 64,
+            "batch_summary_timelimit": 24*60*60,
```

Comment:
Because the API itself has a timelimit of 24h. I just matched it. 
Also, if we reach the timelimit, paperqa raises an exception that is not caught by the test.

Reply:
GitHub Actions only run a step (in this case, our `pytest` step) for 6 or 12 hrs by default, so I think 24 hours is too long